front/components/assistant_builder/Sharing.tsx

```diff
@@ -166,114 +166,123 @@ export function SharingButton({
-                    </div>
-                    <div className="text-sm text-element-700">
-                      {slackChannelSelected.length === 0 ? (
-                        <>
-                          Set as default assistant for specific&nbsp;channels.
-                        </>
-                      ) : (
-                        <>
-                          Default assistant for{" "}
-                          {slackChannelSelected
-                            .map((c) => c.slackChannelName)
-                            .join(", ")}
-                        </>
-                      )}
+            </DropdownMenu.Button>
+            <DropdownMenu.Items width={300} overflow="visible">
+              <div className="flex flex-col gap-y-2 py-1">
+                <div className="flex flex-col gap-y-3">
+                  <SharingDropdown
+                    owner={owner}
+                    agentConfiguration={agentConfiguration}
+                    initialScope={initialScope}
+                    newScope={newScope}
+                    setNewScope={setNewScope}
+                  />
+                  <div className="text-sm text-element-700">
+                    <div>
+                      {SCOPE_INFO[newScope].text}{" "}
+                      {agentUsage && newScope !== "private" ? usageText : null}
                     </div>
+                  </div>
+                </div>
+                {showSlackIntegration && (
+                  <>
+                    <Page.Separator />
+                    <div className="flex flex-row justify-between">
+                      <div>
+                        <div className="text-base font-bold text-element-800">
+                          Slack integration
+                        </div>
+                        <div className="text-sm text-element-700">
+                          {slackChannelSelected.length === 0 ? (
+                            <>
+                              Set as default assistant for
+                              specific&nbsp;channels.
+                            </>
+                          ) : (
+                            <>
+                              Default assistant for{" "}
+                              {slackChannelSelected
+                                .map((c) => c.slackChannelName)
+                                .join(", ")}
+                            </>
+                          )}
+                        </div>
 
-                    {slackChannelSelected.length > 0 && (
-                      <div className="pt-3">
+                        {slackChannelSelected.length > 0 && (
+                          <div className="pt-3">
+                            <Button
+                              size="xs"
+                              variant="secondary"
+                              label="Manage channels"
+                              onClick={() => {
+                                close();
```

Comment:
where does this close() comes from ?

Reply:
It's our dropdown API: https://github.com/dust-tt/dust/blob/53d4b310cf4385e395abe2254c1166659f6e3aae/sparkle/src/stories/DropdownMenu.stories.tsx#L183-L184